src/main/java/kitchenpos/menu/domain/MenuProducts.java

```diff
@@ -26,14 +25,9 @@ public List<MenuProduct> getList() {
     public BigDecimal getSumPrice() {
         BigDecimal sum = BigDecimal.ZERO;
         for (MenuProduct menuProduct : menuProducts) {
-            sum = sum.add(menuProduct.getProduct()
-                    .getPrice().multiply(BigDecimal.valueOf(menuProduct.getQuantity())));
+            sum = sum.add(menuProduct.price());
         }
 
         return sum;
     }
```

Comment:
```suggestion
    public Price getSumPrice() {
        menuProducts.stream()
                .map(MenuProduct::price)
                .reduce(Price::sum)
                .orElseGet(Price.ZERO);
    }
```
합계를 계산하는 책임도 Price에서 담당하면 좋을것같네요 😀
위 코드는 참고정도만 해주세요.